enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/AuthProceduresTest.java

```diff
@@ -406,6 +408,127 @@ public void roleManagement3() throws Exception
         testSuccessfulReadAction( subject, 4L );
     }
 
+    //----------User deletion -----------
+
+    @Test
+    public void shouldDeleteUser() throws Exception
+    {
+        testCallEmpty( db, adminSubject, "CALL dbms.createUser('Craig', '1234', true)", null );
+        assertNotNull( "User Craig should exist", manager.getUser( "Craig" ) );
+        testCallEmpty( db, adminSubject, "CALL dbms.deleteUser('Craig')", null );
```

Comment:
I think this shouldn't fail silently but result in an error, just like creating a user that already exists.